workshop/exercise-3/README.md

```diff
@@ -11,91 +11,103 @@ Then go to the example:
   ```sh
   cd v2
   ```
-  
-## Create Redis backend
-The Redis backend provides the persistance service to the application. It consists of the master and slave modules. We will create the controllers and service for both master and slave.
+
+### Create Redis backend
+The Redis backend provides the persistence service to the application. It consists of the master and slave modules.
+
+1. Create the controllers and service for both master and slave.
   ``` sh
   kubectl create -f redis-master-deployment.yaml
   kubectl create -f redis-master-service.yaml
   kubectl create -f redis-slave-deployment.yaml
   kubectl create -f redis-slave-service.yaml
   ```
-To verify the installation, first we check the controllers:
+2. Check the controllers.
   ```sh
   kubectl get deployment
   NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
   redis-master   1         1         1            1           5d
   redis-slave    2         2         2            2           5d
   ```
-Then we check the service:
+3. Check the service.
   ```sh
   kubectl get svc
   NAME           TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)        AGE
   redis-master   ClusterIP      172.21.85.39    <none>          6379/TCP       5d
   redis-slave    ClusterIP      172.21.205.35   <none>          6379/TCP       5d
   ```
-Lastly we check the pods:
+4. Check the pods.
   ```sh
   kubectl get pods
   NAME                            READY     STATUS    RESTARTS   AGE
   redis-master-4sswq              1/1       Running   0          5d
   redis-slave-kj8jp               1/1       Running   0          5d
   redis-slave-nslps               1/1       Running   0          5d
   ```
-## Install guestbook app with Istio
-
-  ```sh
- kubectl apply -f <(istioctl kube-inject -f ../v1/guestbook-deployment.yaml --debug)
- kubectl apply -f <(istioctl kube-inject -f guestbook-deployment.yaml --debug)
-  ```
-These commands will inject the Istio envoy sidecar into the guestbook pods, as well as deploy the guestbook app on to the K8s cluster. Here we have two versions of deployments, a new version (`v2`) in the current directory, and a previous version (`v1`) in a sibling directory. They will be used in future sections to showcase the Istio traffic routing capabilities.
-  
-Next, we'll create the guestbook service.
+### Install guestbook app with Istio
 
-    kubectl create -f guestbook-service.yaml
+1. Inject the Istio envoy sidecar into the guestbook pods and deploy the guestbook app on to the Kubernetes cluster.
+```sh
+kubectl apply -f ../v1/guestbook-deployment.yaml --debug
+kubectl apply -f guestbook-deployment.yaml --debug
+```
```

Comment:
So the previous commands are for manual sidecar injection. I could only get the above commands ^^ to work because we're using automatic sidecar injection.